[BUG] 参数校验问题导致上传失败

## Describe the bug
添加主播后，录制后上传时出现问题，upload-20250416.log的内容为
```
First upload this live
Current account: -
Current level: 4
Status: 非大会员
probe_version=20221109&upcdn=txa&zone=cs 0.31480128699331544
probe_version=20221109&upcdn=alia&zone=cs 0.8767601040017325
Uploading video: 100%|██████████| 1.46G/1.46G [01:52<00:00, 13.0MB/s]
参数错误
[ERROR] - [2025-04-16 14:01:00,114 bilive upload] - Fail to upload, the files will be locked.
```
参数错误的具体信息没有，所以不知道是什么地方出了问题，bilive.toml的内容为
```
# bilive Settings
[model]
model_type = "append" 
[asr]
asr_method = "none" 

[video]
# You can change the title as you like, eg.
title = "{date}{artist}直播" # Key words: {artist}, {date}, {title}, {source_link}
description = "0" # Key words: {artist}, {date}, {title}, {source_link}
tid = 65 # The tid of the video(int), see https://bilitool.timerring.com/tid.html
gift_price_filter = 1 # The gift whose price is less than this value will be filtered, unit: RMB
reserve_for_fixing = false # If encounter MOOV crash error, delete the video or reserve for fixing
upload_line = "auto" # The upload line to be used, default None is auto detect(recommended), if you want to specify, it can be "bldsa", "ws", "tx", "qn", "bda2".

[slice]
auto_slice = false 
```


## How To Reproduce
从直播推荐首页随机寻找一个主播录制，视频是没有问题正常上传的。出现问题是一个小主播房间为-

## Additional context
另外还想请问一下如何设置不录制字幕，在其他的issue里看到您提到在append部分写了不录制字幕的设置方法，但没有找到

--- a/src/upload/extract_video_info.py
+++ b/src/upload/extract_video_info.py
@@ -76,7 +76,7 @@ def generate_tag(video_path):
     tags = get_bilibili_suggestions(artist_text)
 
     if not tags:
-        tags = ["直播回放", "切片"]
+        tags = "直播回放"
 
     return tags
 

--- a/src/upload/query_search_suggestion.py
+++ b/src/upload/query_search_suggestion.py
@@ -26,10 +26,10 @@ def get_bilibili_suggestions(term):
             f"Request get_bilibili_suggestions failed with status code: {response.status_code}"
         )
         # fallback to default tags
-        return ["直播回放", "切片"]
+        return "直播回放"
     except requests.RequestException as e:
         upload_log.error(f"Request get_bilibili_suggestions failed with exception: {e}")
-        return ["直播回放", "切片"]
+        return "直播回放"
 
 
 if __name__ == "__main__":

